test/components/remote_retro_test.js

```diff
@@ -8,17 +10,22 @@ const { IDEA_GENERATION, CLOSED } = STAGES
 
 describe("RemoteRetro component", () => {
   const mockRetroChannel = {}
-  const stubUser = { given_name: "Mugatu" }
+  const stubUser = {
+    given_name: "Mugatu",
+    is_facilitator: false,
+  }
   const defaultProps = {
     currentUser: stubUser,
     retroChannel: mockRetroChannel,
     isTabletOrAbove: true,
     presences: [],
-    actions: {},
     ideas: [],
     stage: IDEA_GENERATION,
     facilitatorName: "Daniel Handpan",
     retro: { stage: IDEA_GENERATION },
+    actions: {
+      newFacilitator: spy(),
```

Comment:
gonna remove the spy from default props, as this gets applied to all of the tests, and shift spy declaration down into a `beforeEach` so that the spies reset